frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.test.tsx

```diff
@@ -155,3 +179,38 @@ describe('preview with option interactions', () => {
     expect(within(preview).getByText('posts')).toBeInTheDocument()
   })
 })
+
+describe('go to ERD with option select', () => {
+  it('go to the table of clicked option and close dialog', async () => {
+    const {
+      user,
+      elements: { dialog },
+      testElements: { activeTableNameDisplay },
+    } = await prepareCommandPalette()
+
+    expect(activeTableNameDisplay).toBeEmptyDOMElement()
+
+    await user.click(within(dialog).getByRole('option', { name: 'follows' }))
+
+    expect(dialog).not.toBeInTheDocument()
+    expect(activeTableNameDisplay).toHaveTextContent(/^follows$/)
+  })
+
+  it('go to the table of selected option by typing Enter key and close dialog', async () => {
+    const {
+      user,
+      elements: { dialog, preview },
+      testElements: { activeTableNameDisplay },
+    } = await prepareCommandPalette()
+
+    expect(activeTableNameDisplay).toBeEmptyDOMElement()
+
+    // select "posts" option by typing Enter key
+    await user.keyboard('{ArrowDown}')
+    expect(within(preview).getByText('posts')).toBeInTheDocument()
+    await user.keyboard('{Enter}')
+
+    expect(dialog).not.toBeInTheDocument()
+    expect(activeTableNameDisplay).toHaveTextContent(/^posts$/)
```

Comment:
This test covers the following scenario:

1. Assert that no table is active.
1. Select the "post" option using the keyboard.
1. Assert that the dialog is closed and the "post" table is active.